SCClassLibrary/Common/Control/Engine.sc

```diff
@@ -291,26 +295,26 @@ ContiguousBlockAllocator {
 
 	removeFromFreed { |block|
 		freed[block.size].tryPerform(\remove, block);
-			// I tested without gc; performance is about half as efficient without it
+		// I tested without gc; performance is about half as efficient without it
 		(freed[block.size].size == 0).if({ freed.removeAt(block.size) });
 	}
 
 	findPrevious { |address|
 		forBy(address-1, pos, -1, { |i|
-			array[i].notNil.if({ ^array[i] });
+			array[i - addrOffset].notNil.if({ ^array[i - addrOffset] });
 		});
 		^nil
 	}
 
 	findNext { |address|
-		var	temp;
-		(temp = array[address]).notNil.if({
-			^array[temp.start + temp.size]
-		}, {
+		var	temp = array[address - addrOffset];
```

Comment:
these are all inherited from old code, and hard to see in the IDE, but fine, I'll get rid of the other ones I see as well.